packages/azure-themes/src/azure/styles/Link.styles.ts

```diff
@@ -18,6 +18,9 @@ export const LinkStyles = (props: ILinkStyleProps): Partial<ILinkStyles> => {
           '&:active': {
             color: semanticColors.linkHovered,
           },
+          '&:focus': {
+            outline: '0 !important', //outline and border cannot both be active or border looks 2x width on focus. There is another target that has more specificity (.ms-Fabric--isFocusVisible .root-663:focus) that I am unable to target from here, therefore had to use important.
```

Comment:
This is unrelated to the bug in question, but I added to PR since I discovered discrepancy. There is unfortunately a higher CSS specificity that is being used on Links which applies an outline AND a border, and I am unable to achieve higher specificity within the ts stylesheet for Links because of the .ms-Fabric--isFocusVisible being applied at a higher level in the DOM. For now, I am using !important and will consult with Azure engineers to ensure that they are not applying other outline styles to a focused link that this would interfere with.